xbmc/video/windows/GUIWindowVideoBase.h

```diff
@@ -43,6 +43,8 @@ class CGUIWindowVideoBase : public CGUIMediaWindow, public IBackgroundLoaderObse
   virtual ~CGUIWindowVideoBase(void);
   virtual bool OnMessage(CGUIMessage& message) override;
   virtual bool OnAction(const CAction &action) override;
+  virtual void OnInitWindow() override;
+  virtual void OnDeinitWindow(int nextWindowID) override;
```

Comment:
You don't need to specify virtual when using override, override implies virtual. You don’t need to change the pr, just meant as a tip for the future :)